R/scale-colour.r

```diff
@@ -4,22 +4,36 @@
 #' `ggplot2.continuous.colour` and `ggplot2.continuous.fill` options. If these
 #' options are not present, `"gradient"` will be used. See [options()] for more
 #' information.
-#' 
+#'
 #' @param ... Additional parameters passed on to the scale type
 #' @param type One of "gradient" (the default) or "viridis" indicating the
 #'   colour scale to use
 #' @seealso [scale_colour_gradient()], [scale_colour_viridis_c()],
 #'   [scale_fill_gradient()], and [scale_fill_viridis_c()]
 #' @export
 #' @rdname scale_colour_continuous
+#' @section Color Blindness:
+#' Many color palettes derived from RGB combinations (like the "rainbow" color
+#' palette) are not suitable to support all viewers, especially those with
+#' color vision deficiencies. Using `viridis` type, which is perceptually
+#' uniform in both colour and black-and-white display is an easy option to
+#' ensure good perceptive properties of your visulizations.
+#' The colorspace package offers functionalities
+#' - to generate color palettes with good perceptive properties,
+#' - to analyse a given color palette, like emulating color blindness,
+#' - and to modify a given color palette for better perceptivity.
+#'
+#' For more information on color vision deficiencies and suitable color choices
+#' see the [paper on the colorspace package](https://arxiv.org/abs/1903.06490)
+#' and references therein.
```

Comment:
Duplicate code from `scale-manual.r`
Considered using [roxygen templates](https://stackoverflow.com/questions/15100129/using-roxygen2-template-tags), but decided against it because of overhead.